src/mpid/ch4/shm/posix/posix_recv.h

```diff
@@ -41,6 +41,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_POSIX_mpi_recv_init(void *buf,
                                                        MPIR_Comm * comm, int context_offset,
                                                        MPIR_Request ** request)
 {
+
```

Comment:
Hmm, did I include this?

Reply:
possibly my mistake in rebasing. can you clean it and the other comment up? I'm otherwise fine with this as-is.